Readability.js

```diff
@@ -1743,12 +1743,15 @@ Readability.prototype = {
         return false;
       }
 
-      var textContentLength = node.textContent.trim().length;
-      if (textContentLength < 140) {
-        return false;
+      // In some web pages, they use "<br>" to typeset articles.
+      if (node.tagName === "BR") {
+        score += 2;
       }
 
-      score += Math.sqrt(textContentLength - 140);
+      var textContentLength = node.textContent.trim().length;
+      if (textContentLength > 140) {
```

Comment:
Why the change compared to how this length code worked previously? I don't think it really matters in practice (ie I think the outwardly visible behaviour will be the same), but we might as well leave it the same to avoid polluting blame. :-)